ramhorns/src/content.rs

```diff
@@ -403,10 +485,11 @@ impl<T: Content> Content for Vec<T> {
             .render_section(section, encoder)?;
         }
         #[cfg(not(feature = "indexes"))]
-        for item in self.iter() {
-            item.render_section(section, encoder)?;
+        {
```

Comment:
Oh this is leftover from a little bit of debugging I did, so that my prints and the for loop would all be included in the `cfg`.

It's not necessary, but also won't hurt.